src/containers/RequestCodesPage/index.jsx

```diff
@@ -1,18 +1,26 @@
 import { connect } from 'react-redux';
 
 import RequestCodesPage from '../../components/RequestCodesPage';
-
+import { requestCodes, clearRequestCodes } from '../../data/actions/codes';
 import { fetchPortalConfiguration } from '../../data/actions/portalConfiguration';
 
 const mapStateToProps = state => ({
   enterpriseName: state.portalConfiguration.enterpriseName,
   emailAddress: state.authentication.email,
+  error: state.codes.error,
+  success: state.codes.success,
 });
 
 const mapDispatchToProps = dispatch => ({
   fetchPortalConfiguration: (slug) => {
     dispatch(fetchPortalConfiguration(slug));
   },
+  requestCodes: (enterpriseName, emailAddress, numberOfCodes) => {
```

Comment:
As described in another comment, we should pass in a single object here instead of separate arguments. The `requestCodes` action handler can then use object destructuring to extract the 3 values as shown in the example in the other comment.

```javascript
requestCodes: (options) => {
  dispatch(requestCodes(options));
}
```